helm/tailing-sidecar-operator/templates/resources.yaml

```diff
@@ -193,6 +193,22 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
+  verbs:
+  - create
+- apiGroups:
+  - coordination.k8s.io
+  resourceNames:
+  - 7b555970.sumologic.com
```

Comment:
Where does this name come from?

Reply:
https://github.com/SumoLogic/tailing-sidecar/blob/v0.5.1/operator/main.go#L67 🤷

Reply:
in history I see that this `LeaderElectionID` was generated by operator sdk